pkgs/applications/emulators/yuzu/generic.nix

```
@@ -146,9 +150,10 @@ stdenv.mkDerivation rec {
     license = with licenses; [
       gpl3Plus
       # Icons
```

I think we should explain here as well that the license is normally different.
```suggestion
      # Icons. Note that this would be cc0 and cc-by-nd-30 without the "yuzu-free-icons" patch
```
or something like that